src/lib/y2network/config_writer/sysconfig.rb

```diff
@@ -97,6 +96,43 @@ def write_ipv6_forwarding(forward_ipv6)
 
         Yast::SCR.Execute(Yast::Path.new(".target.bash"), "/usr/sbin/sysctl -w #{IPV6_SYSCTL}=#{sysctl_val.shellescape}") == 0
       end
+
+      # Finds routes for a given interface
+      #
+      # @param iface  [Interface,:any] Interface to search routes for
+      # @param routes [Array<Route>] List of routes to search in
+      # @return [Array<Route>] List of routes for the given interface
+      #
+      # @see #find_routes_for_any
+      # @see #find_routes_for_iface
+      def find_routes_for(iface, routes)
+        iface == :any ? find_routes_for_any(routes) : find_routes_for_iface(iface, routes)
+      end
+
+      # Finds routes which are not tied to an specific interface
+      def find_routes_for_any(routes)
+        routes.select { |r| r.interface == :any }
```

Comment:
hmm, more and more I see this `:any` everywhere for route, maybe we can have specific Interface instance for any with its method?